graphrag/callbacks/global_search_callbacks.py

```diff
@@ -3,9 +3,10 @@
 
 """GlobalSearch LLM Callbacks."""
 
-from graphrag.query.llm.base import BaseLLMCallback
 from graphrag.query.structured_search.base import SearchResult
 
+from .llm_callbacks import BaseLLMCallback
```

Comment:
can we use an absolute import here to match the one above it?

Reply:
(not a fan of relative imports and mixing them) - but they sometimes resolve circular dependencies so if it's possible to be consistent it would be great :)